caffe2/operators/quantized/int8_resize_nearest_op.h

```diff
@@ -17,6 +17,7 @@ class Int8ResizeNearestOp final : public Operator<CPUContext> {
       : Operator<CPUContext>(std::forward<Args>(args)...) {
     width_scale_ = this->template GetSingleArgument<float>("width_scale", 1);
     height_scale_ = this->template GetSingleArgument<float>("height_scale", 1);
+    output_dims = this->template GetRepeatedArgument<int>("output_size");
```

Comment:
let's give it a default value, if this is missing, we probably should not crash.